packages/ecs-agent/ecs-agent.spec

```diff
@@ -248,6 +252,27 @@ install -D -p -m 0644 %{S:102} %{buildroot}%{_cross_tmpfilesdir}/ecs.conf
 install -D -p -m 0644 %{S:103} %{buildroot}%{_cross_sysctldir}/90-ecs.conf
 install -D -p -m 0644 %{S:104} %{buildroot}%{_cross_templatedir}/ecs.config
 
+# Directory for agents used by the ECS agent, e.g. SSM, Service Connect
+%global managed_agents %{_cross_libexecdir}/amazon-ecs-agent/managed-agents
+install -d %{buildroot}%{managed_agents}
+
+# Directory for ECS exec artifacts
+%global ecs_exec_dir %{managed_agents}/execute-command
+install -d %{buildroot}%{ecs_exec_dir}
+
+# The ECS agent looks for real versioned directories under bin, symlinks will be
+# ignored. Thus, link the bin directory in the ssm-agent directory which contains
+# the versioned binaries.
+ln -s %{_cross_libexecdir}/amazon-ssm-agent/bin %{buildroot}/%{ecs_exec_dir}/bin
```

Comment:
Can we use a relative symlink here? It is best to avoid full paths in symlinks for packages. If not possible, please add a comment explaining why.

```suggestion
# The ECS agent looks for real versioned directories under bin, symlinks will be
# ignored. Thus, link the bin directory in the ssm-agent directory which contains
# the versioned binaries.
ln -rs %{buildroot}%{_cross_libexecdir}/amazon-ssm-agent/bin %{buildroot}%{ecs_exec_dir}/bin
```